Yoast/wordpress-seo
# Make the Copy home meta description a button and add help.

## Summary

This PR can be summarized in the following changelog entry:

* Improves semantics and accessibility of the "Copy home meta description" button in the Social, Facebook setting, and adds inline Help.

## Relevant technical choices:

- the link is now a button
- removed the inline JS
- added an inline Help panel
- needs #8894 for proper styling

## Test instructions

- build CSS and JS
- chec the button is not displayed at all when the homepage meta description is empty
- when is not empty, check the button works correctly
- check the functionality of the Help panel (this needs #8894 to work properly)
- check the copy used for the Help button and panel, any feedback and improvements welcome

With the changes from #8894, it will look like in the screenshots below:

<img width="651" alt="screen shot 2018-02-14 at 15 28 13" src="https://user-images.githubusercontent.com/1682452/36210993-3460ef9e-11a0-11e8-9870-dc6801d57e82.png">

<img width="675" alt="screen shot 2018-02-14 at 15 28 27" src="https://user-images.githubusercontent.com/1682452/36211170-cba6f5d8-11a0-11e8-819c-72fd221162d0.png">


Fixes #8921
